com.ibm.wala.core/src/main/java/com/ibm/wala/ipa/callgraph/impl/Util.java

```diff
@@ -119,6 +123,47 @@ public static void addBypassLogic(
     }
   }
 
+  /**
+   * Removes some overhead from the previous method, removes the need to pass in an AnalysisScope
+   * parameter
+   */
+  public static void addBypassLogic(
+      AnalysisOptions options, ClassLoader cl, String xmlFile, IClassHierarchy cha)
+      throws IllegalArgumentException {
+    if (cha.getScope() == null) {
+      throw new IllegalArgumentException("scope is null");
+    }
```

Comment:
No need for this check anymore, since if `cha != null`, then `cha.getScope()` won't be null